<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArcadeModeSetup</class>
 <widget class="QWidget" name="ArcadeModeSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arcade mode setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSettings">
      <attribute name="title">
       <string>Global settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxArcadeTheme">
         <property name="toolTip">
          <string>Select arcade theme</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">ToxicWaste</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelGraphicsSystem">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Graphics system</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxGraphicsSystem">
         <property name="toolTip">
          <string>Select the graphics system used to render the scene (when available, OpenGL is strongly recommended!)</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">raster</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">opengl</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">native</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelWorkingDirectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Working directory</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelArcadeTheme">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Arcade theme</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditWorkingDirectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Working directory used when executing QMC2 Arcade</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QComboBox" name="comboBoxConsoleType">
         <property name="toolTip">
          <string>Select type of console</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">terminal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">window</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">window-minimized</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="toolButtonBrowseWorkingDirectory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Browse working directory used when executing QMC2 Arcade</string>
         </property>
         <property name="icon">
          <iconset resource="qmc2.qrc">
           <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="toolButtonBrowseExecutableFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Browse QMC2 Arcade executable file</string>
         </property>
         <property name="icon">
          <iconset resource="qmc2.qrc">
           <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelConsoleType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Console type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEditExecutableFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>QMC2 Arcade executable file</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelExecutableFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Executable file</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelConfigurationPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Configuration path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditConfigurationPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Configuration path for QMC2 Arcade</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="toolButtonBrowseConfigurationPath">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Browse configuration path for QMC2 Arcade</string>
         </property>
         <property name="icon">
          <iconset resource="qmc2.qrc">
           <normaloff>:/data/img/fileopen.png</normaloff>:/data/img/fileopen.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFilter">
      <attribute name="title">
       <string>Game list filter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelROMStatesToSelect">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ROM state filter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButtonSelectC">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select ROM state C (correct)?</string>
           </property>
           <property name="icon">
            <iconset resource="qmc2.qrc">
             <normaloff>:/data/img/sphere_green.png</normaloff>:/data/img/sphere_green.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelectM">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select ROM state M (mostly correct)?</string>
           </property>
           <property name="icon">
            <iconset resource="qmc2.qrc">
             <normaloff>:/data/img/sphere_yellowgreen.png</normaloff>:/data/img/sphere_yellowgreen.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelectI">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select ROM state I (incorrect)?</string>
           </property>
           <property name="icon">
            <iconset resource="qmc2.qrc">
             <normaloff>:/data/img/sphere_red.png</normaloff>:/data/img/sphere_red.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelectN">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select ROM state N (not found)?</string>
           </property>
           <property name="icon">
            <iconset resource="qmc2.qrc">
             <normaloff>:/data/img/sphere_grey.png</normaloff>:/data/img/sphere_grey.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonSelectU">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Select ROM state U (unknown)?</string>
           </property>
           <property name="icon">
            <iconset resource="qmc2.qrc">
             <normaloff>:/data/img/sphere_blue.png</normaloff>:/data/img/sphere_blue.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDriverStatus">
         <property name="text">
          <string>Driver status</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxDriverStatus">
         <property name="toolTip">
          <string>Choose the driver maturity states for sets to be included</string>
         </property>
         <item>
          <property name="text">
           <string>good</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>good + preliminary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>good + preliminary + imperfect</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelNameFilter">
         <property name="text">
          <string>Name filter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditNameFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Specify a name (ID) filter-expression (Qt regular-expression syntax!) -- leave empty to not filter by name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="toolButtonClearNameFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Clear name filter</string>
         </property>
         <property name="icon">
          <iconset resource="qmc2.qrc">
           <normaloff>:/data/img/broom.png</normaloff>:/data/img/broom.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="labelCategoryFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Category filter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QListWidget" name="listWidgetCategoryFilter">
         <property name="toolTip">
          <string>Select / deselect the categories you want to include / exclude</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKeys">
      <attribute name="title">
       <string>Keys</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabJoystick">
      <attribute name="title">
       <string>Joystick</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>82</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>&amp;Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEditExecutableFile</tabstop>
  <tabstop>toolButtonBrowseExecutableFile</tabstop>
  <tabstop>lineEditWorkingDirectory</tabstop>
  <tabstop>toolButtonBrowseWorkingDirectory</tabstop>
  <tabstop>lineEditConfigurationPath</tabstop>
  <tabstop>toolButtonBrowseConfigurationPath</tabstop>
  <tabstop>comboBoxGraphicsSystem</tabstop>
  <tabstop>comboBoxArcadeTheme</tabstop>
  <tabstop>comboBoxConsoleType</tabstop>
  <tabstop>toolButtonSelectC</tabstop>
  <tabstop>toolButtonSelectM</tabstop>
  <tabstop>toolButtonSelectI</tabstop>
  <tabstop>toolButtonSelectN</tabstop>
  <tabstop>toolButtonSelectU</tabstop>
  <tabstop>comboBoxDriverStatus</tabstop>
  <tabstop>lineEditNameFilter</tabstop>
  <tabstop>toolButtonClearNameFilter</tabstop>
  <tabstop>listWidgetCategoryFilter</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonClearNameFilter</sender>
   <signal>clicked()</signal>
   <receiver>lineEditNameFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
